### PR TITLE
Add filesystem offloader configurations in sn-platform charts

### PR DESCRIPTION
**Pull request by zymap**
Fixes: #382 

---

*Motivation*

Allow enabling the filesystem offloader with helm charts.

*Modifications*

Add filesystem offloader configurations in the pulsar charts.

### CODE REVIEW

**Comment by hangc0276** [comment]
https://github.com/streamnative/charts/pull/362

File: charts/sn-platform/templates/broker/broker-statefulset.yaml [hunk 1]
```diff
@@ -297,6 +297,7 @@ spec:
           {{- include "pulsar.broker.runtime.volumeMounts" . | nindent 10 }}
           {{- include "pulsar.broker.offload.volumeMounts" . | nindent 10 }}
           {{- include "pulsar.function.worker.config.volumeMounts" . | nindent 10 }}
+          {{- include "pulsar.broker.offload.filesystem.config.volumeMounts" . | nindent 10 }}
```
Comment (tuteng) [inline charts/sn-platform/templates/broker/broker-statefulset.yaml:300]: If filesystem is set to `false`, do you still need to include this mount?

Comment (zymap) [reply charts/sn-platform/templates/broker/broker-statefulset.yaml:300]: no, already handle that in the broker.tpl

**Comment by fantapsody** [comment]
@tuteng Shall we close this one?

**Pull request closed without merging**